automation/test.sh

```diff
@@ -155,6 +163,23 @@ if [ -n "${KUBEVIRT_CACHE_FROM}" ]; then
 fi
 
 make cluster-down
+
+# Create .bazelrc to use remote cache
+cat >.bazelrc <<EOF
+startup --host_jvm_args=-Dbazel.DigestFunction=sha256
+build --remote_local_fallback
+build --remote_http_cache=http://bazel-cache.kubevirt-prow.svc.cluster.local:8080/kubevirt.io/kubevirt
+EOF
+
+# build all images with the basic repeat logic
+set +e
```

Comment:
Can you add a comment here which explains why we do this crazy thing?

Reply:
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/7657/pull-kubevirt-e2e-arm64/1520646845313847296

Wiping the greenhouse cache doesn't help. Looking further for a root casue/workaround